jackal_control/launch/control.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
-  <rosparam command="load" file="$(find jackal_control)/config/control.yaml" />
+  <rosparam file="$(find jackal_control)/config/control.yaml" command="load"/>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="jackal_joint_publisher jackal_velocity_controller --shutdown-timeout 1" />
-  
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" /> 
+
+  <include file="$(find jackal_control)/launch/robot_localization.launch"/>
```

Comment:
First of all, I don't really need this anymore.

That being said, I think the changes pending are minimal... I can look into that if I have some time this weekend.